node/src/bootstrapper.rs

```diff
@@ -400,7 +400,7 @@ impl BootstrapperConfig {
             neighborhood_config: NeighborhoodConfig {
                 mode: NeighborhoodMode::ZeroHop,
                 min_hops: DEFAULT_MIN_HOPS,
-                country: "ZZ".to_string(),
+                exit_locations_opt: None,
```

Comment:
If we're always going to establish exit conditions via the UI once the Node is running, then this information doesn't need to be in the `NeighborhoodConfig`, and doesn't do any good here. It's only useful if we're going to accept exit conditions via command-line parameters.

However, wherever `exit_locations_opt` is, `fallback_routing` should also be. (Perhaps they should both be in a single structure!)